src/manage_nsfs/manage_nsfs_anonymous.js

```diff
@@ -0,0 +1,82 @@
+/* Copyright (C) 2020 NooBaa */
+'use strict';
+
+const _ = require('lodash');
+const native_fs_utils = require('../util/native_fs_utils');
+const mongo_utils = require('../util/mongo_utils');
+const ManageCLIError = require('../manage_nsfs/manage_nsfs_cli_errors').ManageCLIError;
+const ManageCLIResponse = require('../manage_nsfs/manage_nsfs_cli_responses').ManageCLIResponse;
+const nsfs_schema_utils = require('../manage_nsfs/nsfs_schema_utils');
+const { ACTIONS} = require('../manage_nsfs/manage_nsfs_constants');
+const { throw_cli_error, write_stdout_response, get_config_file_path,
+    get_config_data} = require('../manage_nsfs/manage_nsfs_cli_utils');
+const { validate_anonymous_account_args } = require('../manage_nsfs/manage_nsfs_validations');
+
+
```

Comment:
also, anonymous account can not create buckets, is that correct? this is another difference in the configs

Reply:
@romayalon Both anon and normal account dont have that much commen inside add/update/delete methods, that because of the missing access key. These methods have access key and encryption code that is not relevent to anon account and controlling it using the check will add complexity to the code. If you want I can try that.